Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -827,18 +827,41 @@ public final class ManifestLoader: ManifestLoaderProtocol {
                         // This provides some safety against arbitrary code execution when parsing manifest files.
                         // We only allow the permissions which are absolutely necessary.
                         if self.isManifestSandboxEnabled {
-                            let cacheDirectories = [self.databaseCacheDir, moduleCachePath].compactMap{ $0 }
-                            let strictness: Sandbox.Strictness = toolsVersion < .v5_3 ? .manifest_pre_53 : .default
-                            cmd = Sandbox.apply(command: cmd, strictness: strictness, writableDirectories: cacheDirectories)
+                            var sandbox = SandboxProfile()
+                            
+                            // Allow writing inside the temporary directory.
+                            sandbox.pathAccessRules.append(.writable(localFileSystem.tempDirectory))
```

Comment:
hard coding to localFileSystem okay in this case?

Reply:
Yes, because the sandbox is always in terms of the local file system (when the manifest does file system operations, it's always in the context of the local file system).